circle.yml

```diff
@@ -1204,17 +1252,28 @@ jobs:
           repo: cypress-example-recipes
           command: npm run test:ci:firefox
 
+  test-binary-against-recipe-pull-request:
+    <<: *defaults
+    steps:
+      - test-binary-against-repo:
+          repo: cypress-example-recipes
+          command: npm run test:ci
+          pull_request_id: 492
```

Comment:
Why is this hardcoded?

Reply:
because I as a user would say "for this PR run this OTHER PR number in cypress-example-recipes" to show it off